### PR TITLE
ci: add script to create incentives on a liquidity hub

### PR DESCRIPTION
**Pull request by kerber0x**
## Description and Motivation

This PR adds a script to create incentives on a liquidity hub. Handy for deploying incentives in a satellite market for first time.  

## Related Issues

---
## Checklist:

- [x] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-core/blob/main/CONTRIBUTING.md).
- [x] My pull request has a sound title and description (not something vague like `Update index.md`)
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation.
- [ ] The code is formatted properly `cargo fmt --all --`.
- [ ] Clippy doesn't report any issues `cargo clippy -- -D warnings`.
- [ ] I have regenerated the schemas if needed `cargo schema`.

### CODE REVIEW

File: scripts/deployment/create_incentives.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+deployment_script_dir=$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)
```
Comment (0xFable) [inline scripts/deployment/create_incentives.sh:3]: explain BASH_SOURCE, is it just the source of this script ?

File: scripts/deployment/create_incentives.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  case "$arg" in
+  c)
+    chain=$OPTARG
+
+    source $deployment_script_dir/deploy_env/chain_env.sh
+    init_chain_env $OPTARG
+    if [[ "$chain" = "local" ]]; then
+      tx_delay=0.5
+    else
+      tx_delay=8
```
Comment (0xFable) [inline scripts/deployment/create_incentives.sh:95]: something I thought about but not needed now 
We can differentiate with an IF statement based on the platform and if its linux provide 8s, if its mac or something else just 8.

**Review by 0xFable — APPROVED** [review]
LGTM great work!

File: scripts/deployment/create_incentives.sh (hunk 2 above)
Comment (kerber0x) [reply scripts/deployment/create_incentives.sh:95]: I think just the number works on Linux too, need to verify

File: scripts/deployment/create_incentives.sh (hunk 1 above)
Comment (kerber0x) [reply scripts/deployment/create_incentives.sh:3]: That's the path of the script being executed

**Pull request merged**